modules/widgets/wordpress-post-widget.php

```diff
@@ -734,26 +734,30 @@ public function widget( $args, $instance ) {
 		// Enqueue front end assets.
 		$this->enqueue_scripts();
 
-		echo $args['before_widget'];
+		$content = '';
+
+		$content .= $args['before_widget'];
```

Comment:
The initialization of `$content` to `''` is redundant and you can initialize it to `$args['before_widget']`.